careers/application-process/index.html

```diff
@@ -281,13 +281,17 @@ <h3 class="u-hide-on-mobile">Asking About your Application</h3>
     <div class="block
                 block__padded-top
                 block__border-top
+                block__flush-bottom
                 content-l">
-        <section class="content-l_col content-l_col-1-2">
+        <article class="content-l_col content-l_col-1-3">
+            {% include "_template-current-openings.html" %}
+        </article>
+        <article class="content-l_col content-l_col-1-3">
             {% include "_template-working-at-cfpb.html" %}
-        </section>
-        <section class="content-l_col content-l_col-1-2">
+        </article>
```

Comment:
I was going off of this rationale for articles: [`… content in the article element should be independently distributable or reusable.`](https://www.w3.org/wiki/HTML/Elements/article), of which these excerpts seem to be since they are self-contained blocks of content that are used on multiple pages. What do you think?